src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,220 @@
+      background-color: $ui-03;
+      font-weight: 700;
+    }
+
+    tbody {
+      @include type-style('body-short-01');
+      background-color: $ui-01;
+      width: 100%;
+    }
+
+    tr {
+      border: none;
+      height: rem(48px);
+      width: 100%;
+    }
+
+    tbody tr:hover:not(.#{$prefix}--row-disabled) {
+      background: $hover-field;
+    }
+
+    tbody tr:hover:not(.#{$prefix}--row-disabled) td {
+      color: $text-01;
+      background: $hover-field;
+      border-bottom: 1px solid $hover-field;
+      border-top: 1px solid $hover-field;
+    }
+
+    th,
+    td {
+      padding-left: $spacing-sm;
+      padding-right: $spacing-sm;
+      vertical-align: top;
+      text-align: left;
+      color: $text-02;
+      border-top: 1px solid $ui-01;
+      border-bottom: 1px solid $ui-01;
+    }
+
+    th {
+      color: $text-01;
+      border-top: 1px solid $ui-03;
+      border-bottom: 1px solid $ui-03;
+    }
+
+    th:first-of-type:not(.#{$prefix}--table-column-checkbox) {
+      padding-left: $spacing-lg;
+    }
+
+    th.#{$prefix}--table-column-icon:hover,
+    th.#{$prefix}--table-column-menu:hover,
+    th.#{$prefix}--table-expand-v2:hover {
+      background: $ui-03;
+    }
+
+    tbody tr td {
+      background: $ui-01;
+      border-bottom: 1px solid $ui-03;
+      border-top: 1px solid $ui-01;
+    }
+
+    td:not(.#{$prefix}--table-column-checkbox):not(.#{$prefix}--table-column-menu):not(.#{$prefix}--table-column-icon):not(.#{$prefix}--table-expand-v2) {
+      padding: rem(14px) $spacing-sm;
+    }
+
+    td:not(.#{$prefix}--table-column-checkbox):not(.#{$prefix}--table-column-menu):not(.#{$prefix}--table-column-icon):not(.#{$prefix}--table-expand-v2):first-of-type {
+      padding-left: $spacing-lg;
+    }
+
+    td:not(.#{$prefix}--table-column-checkbox):not(.#{$prefix}--table-column-menu):not(.#{$prefix}--table-column-icon):not(.#{$prefix}--table-expand-v2):last-of-type {
+      padding-right: $spacing-lg;
+    }
+
+    .#{$prefix}--table-header-label {
+      display: block;
+      padding: rem(14px) 0;
+    }
+
+    //width for checkbox columns
+    .#{$prefix}--table-column-checkbox {
+      width: rem(50px);
+      min-width: rem(50px);
+      padding: rem(10px) $spacing-md 0 $spacing-md;
+    }
+
+    //width for overflow menu columns
+    .#{$prefix}--table-column-menu {
+      width: rem(52px);
+      min-width: rem(52px);
+      padding-top: rem(8px);
+      padding-right: rem(8px);
+    }
+
+    td.#{$prefix}--table-column-menu .#{$prefix}--overflow-menu[aria-expanded='false']:hover {
+      background: $ui-03;
+    }
+
+    // Checbox and Overflow Menu Overrides
+    .#{$prefix}--checkbox-label {
+      padding-left: rem(28px);
+    }
+
+    td.#{$prefix}--table-column-menu .#{$prefix}--overflow-menu[aria-expanded='false']:focus {
+      @include focus-outline('outline');
+    }
+
+    td.#{$prefix}--table-column-menu .#{$prefix}--overflow-menu[aria-expanded='true']:focus {
+      outline: none;
```

Comment:
Why are we resetting the focus outline here? Should focus be shifting to first menu item?

Reply:
When it's focused and open the bottom outline gets cut off, so I reset it but can find another work around if it should stay focused. Regarding the focus to the first menu item, I didn't change anything from v9 for this. I assumed it was done on purpose since v9 doesn't focus on the first item when it's opened. 
 
<img width="189" alt="Screen Shot 2019-03-20 at 10 24 21 AM" src="https://user-images.githubusercontent.com/32556167/54696910-af030180-4afa-11e9-83c2-89fa9cb656b1.png">

Reply:
The overflow menu should be implementing this pattern (if using `role="menu"`) so wouldn't necessarily be a data table bug.

Reply:
Ah doesn't look like that's in there from v9. I'll add it.

Reply:
hmm.. maybe I'm not understanding correctly. It doesn't seem to be doing anything different.

Reply:
@jnm2377 I think it's just a bug on the overflow menu, might be worth flagging in an issue instead of implementing custom styles 👍